packages/vite/index.d.cts

```diff
@@ -0,0 +1,6 @@
+/**
+ * @deprecated The CJS build of Vite's Node API is deprecated. See https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated for more details.
+ */
+declare const module: any
+
+export = module
```

Comment:
The types should still work even in deprecated state. They should be removed together in v6.

Reply:
Yup there should still work with this from local testing.